hono/migrate.ts

```diff
@@ -2,12 +2,14 @@ import { config } from 'dotenv'
 import { drizzle } from 'drizzle-orm/postgres-js'
 import { migrate } from 'drizzle-orm/postgres-js/migrator'
 import postgres from 'postgres'
+import { itemsExtended } from './src/schema'
 
 config({ path: '.dev.vars' })
 
 const main = async () => {
   const db = drizzle(postgres(process.env.DATABASE_URL!, { ssl: 'require', max: 1 }))
   await migrate(db, { migrationsFolder: './src/drizzle' })
+  await db.refreshMaterializedView(itemsExtended)
```

Comment:
I'm not familiar with Typescript but should the `await migrate` statement finish before the `await db.refreshMaterializedView` statement? I'm not sure if that is already happening here.

Reply:
Yes, this is the expected behavior as `await` is blocking, i.e migrate will run to completion (what we want since we need to configure the db to even run refreshes on a materialized view) then refreshMaterializedView will run to completion.